From 0f963c5e338221f16eb46e98b88ac4692840d864 Mon Sep 17 00:00:00 2001
From: David Sehr <sehr@google.com>
Date: Sat, 25 Apr 2020 15:50:40 -0700
Subject: [PATCH 1/7] sm8350-common: Enable usage of dex2oat64

Use 64-bit dex2oat for better dexopt time.

Bug: 153380900
Test: boot and install an application
Change-Id: I3e7a6e6e9385ff6564d1a2e6dda004ebb061f095
(cherry picked from commit 126f03be80f57a8a0411842011152d9381589b78)
Merged-In: I3e7a6e6e9385ff6564d1a2e6dda004ebb061f095
---
 vendor.prop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/vendor.prop b/vendor.prop
index 3acfb992..86a13941 100644
--- a/vendor.prop
+++ b/vendor.prop
@@ -155,6 +155,9 @@ ro.soc.manufacturer=QTI
 ro.crypto.dm_default_key.options_format.version=2
 ro.crypto.volume.metadata.method=dm-default-key
 
+# Dalvik
+dalvik.vm.dex2oat64.enabled=true
+
 # DPM
 persist.vendor.dpm.feature=1
 persist.vendor.dpm.idletimer.mode=default
-- 
2.49.0

